pkg/config/middleware.go

```diff
@@ -120,6 +128,162 @@ func setupRecoveryMiddleware() *negroni.Recovery {
 	return r
 }
 
+type OidcConfiguation struct {
+	JwkURI string `json:"jwks_uri"` // This is the only field we care about for now.
```

Comment:
any specific reason that the field name is not matching the json field name? i.e.

```
JwksURI string `json:"jwks_uri"`
```